py/circuitpy_mpconfig.mk

```diff
@@ -26,6 +26,18 @@
 # Boards default to all modules enabled (with exceptions)
 # Manually disable by overriding in #mpconfigboard.mk
 
+enable-if-any=$(lastword $(sort $(1) 0))
+enable-if-all=$(firstword $(sort $(1) 1))
```

Comment:
Could you add a comment about how to use these, with `$(call` ? Just an example would be fine.